cmd/kubepkg/main.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import "k8s.io/release/cmd/kubepkg/cmd"
+
+func main() {
```

Comment:
Do we want to add this tool to krel as subcommand?

Reply:
@saschagrunert -- I was thinking it would be better to keep this one separate from `krel`, allowing people who want to use it to do so while minimizing what they pull in from this repo.

Reply:
Okay, sounds good :)